src/app/components/pager/terra-pager.component.ts

```diff
@@ -15,14 +15,14 @@ import { Subject } from 'rxjs/Subject';
     styles:   [require('./terra-pager.component.scss')],
     template: require('./terra-pager.component.html')
 })
-export class TerraPagerComponent implements OnInit
+export class TerraPagerComponent<I extends TerraPagerInterface<I>> implements OnInit
```

Comment:
Ich glaube nicht, dass uns das wirklich was bringt.

Reply:
die Alternative wäre, alle Stellen in der Pager Component, die das TerraPagerInface verwenden mit \<any\> als Type zu nutzen

Reply:
Genau. Und da wir die PagerComponent nirgends per Constructor aufrufen, bzw. als Variablentyp nutzen, können wir eh nichts anderes angeben ;-)